aiida_orca/parsers/cclib/orcaparser.py

```diff
@@ -845,8 +845,8 @@ def splitter(line):
                 self.mosyms.append([])
 
                 line = next(inputfile)
-                while len(line) > 20:  # actually terminated by ------
-                    info = line.split()
+                info = line.split()
+                while len(line) > 20 and len(info) > 2 and len(info) < 6:  # actually terminated by ------
```

Comment:
The original code would fail minimal ORCA output when using the `MINIPRINT` keyword. The fix is not great but it should work.